pkgs/chocolatey/scala.nuspec

```
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>scala</id>
+    <version>@LAUNCHER_VERSION@</version>
+    <title>Scala</title>
+    <authors>scala</authors>
+    <owners>scala</owners>
+    <tags>scala</tags>
+    <summary>Scala</summary>
+    <description>Official release of the Scala Programming Language on Chocolatey.</description>
+    <packageSourceUrl>https://github.com/scala/scala3/tree/main/pkgs/chocolatey</packageSourceUrl>
+    <projectSourceUrl>https://github.com/scala/scala3</projectSourceUrl>
+    <projectUrl>https://scala-lang.org/</projectUrl>
+    <bugTrackerUrl>https://github.com/scala/scala3/issues</bugTrackerUrl>
+    <copyright>© 2002-2024, LAMP/EPFL</copyright>
+    <iconUrl>https://cdn.jsdelivr.net/gh/scala/scala3@version/pkgs/chocolatey/icon.svg</iconUrl>
```

I will have to add the commit hash after we merge it here
```suggestion
    <iconUrl>https://cdn.jsdelivr.net/gh/scala/scala3@<merge-commit>/pkgs/chocolatey/icon.svg</iconUrl>
```